projects/wx-network-wx/index.js

```diff
@@ -0,0 +1,20 @@
+const { assetBalance } = require("../helper/chain/waves");
+const sdk = require('@defillama/sdk')
+
+const WXStakingWxTokenContract = "3PJL8Hn8LACaSBWLQ3UVhctA5cTQLBFwBAP";
+const WXAssetId = "Atqv59EYzjFGuitKVnMRk6H8FukjoV3ktPorbEys25on";
+
+async function WXTVL() {
+  const balances = {};
+  const { balance } = await assetBalance(WXStakingWxTokenContract, WXAssetId);
+  sdk.util.sumSingleBalance(balances,'waves-exchange', balance / 1e8);
+  return balances;
+}
+
+module.exports = {
+  timetravel: false, // Waves blockchain
+  methodology: "TVL of WX means the quantity of staked WX tokens on WX Staking contract",
+  waves: {
+    tvl: WXTVL,
```

Comment:
staking , since its your own token
We dont count own tokens as TVL, should be staking

Reply:
It's users tokens in stake
https://wx.network/governance/wx_staking

Reply:
But is your governance token , so it belong to staking and not TVL 
![image](https://github.com/DefiLlama/DefiLlama-Adapters/assets/93603962/1b10b276-84de-4a66-bfba-aa1aeb3b987d)
https://docs.llama.fi/list-your-project/what-to-include-as-tvl

Reply:
Ok. Thank you.